librz/include/rz_analysis.h

```diff
@@ -737,6 +738,15 @@ typedef struct rz_analysis_var_t {
 	int argnum;
 } RzAnalysisVar;
 
+typedef struct rz_analysis_var_global_t {
```

Comment:
please use doxygen-style doc. And add a simple description for the whole structure as well.

e.g. Use `///<` instead of `//` for struct members.